unix/README.md

```diff
@@ -26,7 +26,11 @@ following packages pre-installed:
     libjpeg8-dev
     libtiff5-dev
     libopenexr-dev
-    
+
+To enable the, as rendering, Simple DirectMedia Layer display also pre-install:
```

Comment:
OK. How about: 

"To enable the Simple DirectMedia Layer display also pre-install:" 

?